source/trunk/Esf.Web/App_Start/RouteConfig.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+using System.Web.Routing;
+
+namespace Esf.Web
```

Comment:
I would name the website project Esf.Website or something need to reflects that this is the website + frontend stuff like typescript css, etc.